public/scripts/ssooverride.js

```diff
@@ -32,28 +45,21 @@
       var profileEl = userInfoDropdown.find('a[data-profile]');
 
       function enable(user) {
+        // XXXbramble - confirm that `user` matches what is
+        // expected here
         joinEl.addClass("hidden");
         loginEl.addClass("hidden");
+        profileEl.addClass("hidden");
 
-        avatarEl.attr("src", user.avatar);
+        avatarEl.attr("src", user.gravatar);
         usernameEl.text(user.username);
-        if (user.isAdmin) {
-          adminEl.removeClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.addClass("hidden");
-        } else if (user.isSuperMentor) {
-          adminEl.addClass("hidden");
-          supermentorEl.removeClass("hidden");
-          mentorEl.addClass("hidden");
-        } else if (user.isMentor) {
-          adminEl.addClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.removeClass("hidden");
-        } else {
-          adminEl.addClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.addClass("hidden");
-        }
+
+        // XXXbramble - until we figure out a permenent UI solution, we're hijacking
```

Comment:
This commend does not tell us anything other than "this code is here because it's here". What part of the following code is a hijack? What needs to be done in the future? Where's the followup bug#? etc.